test/test_solvers.jl

```diff
@@ -1,188 +0,0 @@
-@printf("\nTesting solvers\n")
```

Comment:
Could you just (quickly) make sure these tests are somehow in ProximalAlgorithms? If not, we should have them there at some point

Reply:
yes I think we have very similar stuff in ProximalAlgorithms: 
* [normal lasso](https://github.com/kul-forbes/ProximalAlgorithms.jl/blob/master/test/test_lasso_small.jl)
*  [lasso with multiple variables](https://github.com/kul-forbes/ProximalAlgorithms.jl/blob/master/test/test_lasso_small_split_x.jl) 
* [multiple terms in the smooth functions](https://github.com/kul-forbes/ProximalAlgorithms.jl/blob/master/test/test_lasso_small_split_f.jl). 

maybe some tests like the ones of this file have to be added there as well, e.g. testing subgradients.